modules/monitoring/manifests/checks/cache.pp

```diff
@@ -4,9 +4,15 @@
 #
 # === Parameters
 #
+# [*enabled*]
+#   This variable enables to define whether to perform Icinga Elastic-Cache checks.
+#
 # [*servers*]
 #   List of RDS instances.
 #
+# [*region*]
+#   Which AWS region should be checked ['us-east-1','eu-west-1']
```

Comment:
Should we set the default `region` to `eu-west-1` (Ireland)?

Reply:
Hi @issyl0 , We can. However, I don't think that there is some value on adding here. Also, this might be an additional step for someone searching for the first time to understand.